lib/reverse_sentence.rb

```diff
@@ -2,5 +2,17 @@
 # Time complexity: ?
 # Space complexity: ?
 def reverse_sentence(my_sentence)
```

Comment:
This works, but your space complexity is O(n) because you split the string into an array (creating an array of all the words), further there is an alternative solution which is also O(n) time complexity and O(1) space complexity.

Consider: Reverse the string completely. Then loop through the string reversing each individual word, in place (i.e. not making an array or another string). Think about how you might be able to do that.

I also encourage you to try to analyze your solution for time/space complexity.  This is an important skill for a professional developer.